kinder/pkg/cluster/manager/actions/kubeadm-join.go

```diff
@@ -97,11 +97,6 @@ func kubeadmJoinControlPlane(cp *status.Node, patchesDir, ignorePreflightErrors
 		fmt.Sprintf("--ignore-preflight-errors=%s", ignorePreflightErrors),
 		fmt.Sprintf("--v=%d", vLevel),
 	}
```

Comment:
I cannot comment on Line93, a quick question, does the parameter `patchesDir` is still needed, if it's only read from config file, I think we can remove it from func signature.